change/@microsoft-teams-js-cc1d35f9-c5f8-4d2b-9161-2d3c53d6d593.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Included appNotification.ts file alongside  changes",
```

Comment:
>Included appNotification.ts file alongside  changes [](http://example.com/codeflow?start=14&length=51)

This comment should be a summary of the whole change you are committing and understandable by people reading the changelog. Based on that, I would suggest something like "Added new `appNotification` capability for apps to send notifications to the user through the host"

Reply:
Alright,I'll take note of this for further commits.